content/en/docs/rosenpass-tool/guides/ubuntu.md

```diff
@@ -0,0 +1,275 @@
+---
+aliases: [/docs/guides/ubuntu]
+title: "Rosenpass Installation and Usage on Ubuntu"
+linkTitle: "Ubuntu"
+weight: 3
+menu: false
+type: docs
+shortBlerb: "Set up and use Rosenpass on Ubuntu"
+blerb: "A guide on different options to install the Rosenpass tool on Ubuntu (including installation via binary files and manual compilation), and a usage example as enhancement of WireGuard."
+---
+
+Rosenpass is well tested on Ubuntu. This page provides you with installation and usage guides to get started with Rosenpass on Ubuntu. Currently, most users should start with the [Installation via the Binary Files](#installation-via-binary-files).
+
+Table of Contents:
+* [Installation via the Package Manager](#installation-via-the-package-manager): A short note only, because this is not yet ready for use.
+* [Installation via the Binary Files](#installation-via-binary-files): Most users can start here.
+* [Compile Rosenpass from Source](#compile-it-from-source): Some users might need to go the manual way.
+* [Set up a Rosenpass-enhanced WireGuard VPN](#set-up-a-rosenpass-enhanced-wireguard-vpn): After installation, you can proceed here.
+
+This guide was last tested on March 10, 2025, under Ubuntu 24.10 Mantic Minotaur.
+<span class="spacer"></span>
+
+{{< blocks/lead color="secondary" class="title-box" >}}
+## Installation via the Package Manager
+{{< /blocks/lead >}}
+
+{{% blocks/section color="light" class="no-flex contains-code-snippets package" %}}
+
+**This is not ready, yet!** In the meanwhile, **you are advised to use the [installation via binary files](#installation-via-binary-files)**.
```

Comment:
Couldn't we just add those `.deb` packages? The way they are build right now should be easy enough to backport.

Reply:
Yes, I think we could (and maybe should) do that – although not as part of this task/pull request. We can update the Ubuntu and Debian guides in case we actually get that done.

Reply:
I just created issue #646 to track this.